.travis.yml

```diff
@@ -1,8 +1,9 @@
-language: php
```

Comment:
```
- language: php
```
I think you have to keep this line to enable build against multiple php version ⚠️